docs/SUPPORT.md

```diff
@@ -14,6 +14,8 @@
 
 # Support
 
-For all your support needs or to follow the latest ongoing discussions and updates, join our Slack group. Click [Here](http://del.ly/Slack_request) to request your invite.
+We use GitHub for creating/tracking various [issues](https://github.com/dell/csm/issues/new/choose) such as bugs, feature requests, features, and questions. The following resources are also available.
```

Comment:
References to GH issues can also be removed. We should just point to csm-docs Support page from all the repos. No need to maintain this file.